.github/workflows/release.yml

```diff
@@ -1,22 +1,33 @@
+---
 name: Release
 
 on:
-  create:
-    ref_type: tag
+  push:
+    tags:
+      - '*'
 
 jobs:
   release:
     runs-on: ubuntu-latest
-    if: github.repository == 'voxpupuli/voxpupuli-acceptance'
+    if: github.repository_owner == 'voxpupuli'
     steps:
       - uses: actions/checkout@v2
-      - name: Install Ruby 2.7
+      - name: Install Ruby 3.1
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '2.7'
+          ruby-version: '3.1'
+        env:
+          BUNDLE_WITHOUT: release
```

Comment:
This is not needed since we don't run bundler
```suggestion
```

Reply:
fixed it